### ISSUE TITLE
Overflow Menu focus indicator not visible in high-contrast mode

### ISSUE THREAD

**Issue by johnharan** [IBM/carbon-components-angular#281]
In high-contrast mode the focus indicator is not visible for the overflow menu button.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> What browser are you working in?

Firefox 60.2.1 esr

> What version of the Carbon Design System are you using?

"carbon-components": "9.48.0",
"carbon-components-angular": "2.0.8"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

## Steps to reproduce the issue

1. Tab to the overflow menu button in high contrast mode.
2. Note the focus indicator is not visible.

## Additional information

![2018-11-14_12-20-13](https://user-images.githubusercontent.com/8806230/48621426-6811cd80-e99b-11e8-9dde-9817342221e2.png)
![2018-11-14_12-37-20](https://user-images.githubusercontent.com/8806230/48621427-6811cd80-e99b-11e8-8e0f-a9c54f8a6075.png)

**Comment by youda97** [comment]
What do you mean it is not visible? Its just grey, but still visible.
And if this is still a styling issue it should probably be lifted to https://github.com/IBM/carbon-components

**Comment by johnharan** [comment]
For example, the focus indicator is visible in the below non-high-contrast-mode screenshot
![nonhcwithfocus](https://user-images.githubusercontent.com/8806230/48762961-1c656980-eca4-11e8-8851-830ae0b21e44.png)

Yep wasn't sure if it was a carbon-components issue, have tested using their example table and same issue is visible there, will raise against them, thanks.

**Comment by johnharan** [comment]
closing and adding to carbon-components: https://github.com/IBM/carbon-components/issues/1460
